AElf.Kernel.Tests/Managers/ChainManagerTests.cs

```diff
@@ -1,573 +0,0 @@
-using System;
```

Comment:
why deleted？

Reply:
ChainManagerTests is already exists. somebody add a new one, and commented this before.